bootstrap.sh

```diff
@@ -1,10 +1,77 @@
 #!/bin/bash
 
-set -ueo pipefail
-
 echo "/dev/disk/by-id/google-elasticpd /var/lib/elasticsearch ext4 defaults 0 0" >> /etc/fstab
 lsblk -o label /dev/disk/by-id/google-elasticpd | grep -q elastic-data || mkfs.ext4 -L elastic-data /dev/disk/by-id/google-elasticpd
 mount /var/lib/elasticsearch
 chown elasticsearch:elasticsearch /var/lib/elasticsearch
 
+
+if ! grep 'elasticsearch soft memlock unlimited' /etc/security/limits.conf; then
+
+(
+cat <<ADDTEXT
+elasticsearch soft memlock unlimited
+elasticsearch hard memlock unlimited
+ADDTEXT
+) >> /etc/security/limits.conf;
+
+fi
+
+su elasticsearch -s /bin/bash -c '/usr/share/elasticsearch/bin/elasticsearch | tee /tmp/elasticsearch_bootstrap_start.log >/dev/full' &
+
+echo "waiting for ES7 bootstrap to start"
+while true; do
+  grep "starting GCE discovery service" /tmp/elasticsearch_bootstrap_start.log && break;
+  sleep 10
+done
+
+sleep 10
```

Comment:
Nevím, proč je tu potřeba tenhle `sleep`, ale na první pohled mi to přijde jako další důvod, proč nečekat na základě grepování logu 🙂